### DISCUSSION TITLE
書單分享

### DISCUSSION THREAD

**Discussion by Mitochondrionjessica** [1101-GS4719-C/Python-Community-GS4719-C#1986] [General]
愛看書的人類最怕鬧書荒，下面推薦幾本最近看的書，歡迎大家踴躍分享自己的書單和討論~

1.原子習慣
不論是想建立好習慣，或改掉壞習慣的人都適合看。習慣是堆疊而成的，利用提示、渴望、回應、獎賞這四大階段模型養成你想要有的好習慣，而想改掉壞習慣就要避免提示的產生。

2.為什麼這樣工作 那樣生活
只要我們選擇了我們要成為的人，人們就會朝著他們練習的方式前進，就像一張紙一但被折過，之後就容易折成同樣的折。書裡提到很多知名的企業如何運用習慣，讓自己的事業達到高峰，或是改變危機成轉機。也提到了當時的馬丁路德的黑人人權運動也是運用了社會的習慣才造成相當大的轟動，在在地都顯示著習慣的力量。

3.多巴胺國度
「拼命追逐愉悅（逃避痛苦）只會通往更多痛苦。」有節制地追求愉悅，避免過度依賴某些刺激，才能夠維持大腦對愉悅和痛苦的健康平衡。

4.神探李昌鈺破案實
推理迷必看的經典好書，書中詳細記載刑事鑑識學專家李昌鈺博士經歷，寫作手法高明，帶著讀者身歷其境還原現場。不論是燒腦程度還是緊張懸疑的氣氛都拿捏得恰到好處。

**Comment by YuanWang08** [comment]
你的讀書清單絕對讓喜歡閱讀的朋友大獲裨益。你推薦的書籍涵蓋多元主題，對於想要廣泛獲取知識的讀者來說絕對是福音。關於原子習慣這本書講述了如何通過微小且持續的改變來塑造生活。這絕對是一本不僅有助於理解習慣如何形成，同時也能幫助我們在日常生活中實現長期的目標的絕佳書籍。

**Comment by jylin34** [comment]
四本中只看過原子習慣，作者透過分享自己的經驗，總結出一套幫助人們培養好習慣 促進自我成長的方法。類似原子習慣這種心靈雞湯的書，我覺得 底層邏輯 也不錯。 最近也有看一本"The Big Short 大賣空" 講述08年次貸危機的故事，對金融有興趣的人，我也十分推薦這本書。
